app/views/hyrax/dashboard/profiles/_edit_primary.html.erb

```diff
@@ -19,12 +19,14 @@
     <br>
     <!-- I would like to move this to the bottom of the form but whenever I try the styling is all
     wrong and I don't want to push the pixels around right now-->
+    <% if current_user.has_role?(:admin, Site.instance) %>
      <div class="checkbox">
        <%= f.label :allow_public_display do %>
         <%= f.check_box :display_profile %>
```

Comment:
The label symbol and the checkbox symbol need to match otherwise you'll break some accesibility rules in HTML. 

```
       <%= f.label :public_display do %>
        <%= f.check_box :display_profile %>
```

Reply:
Thanks :)